andela/AndelaEatsUI
# #166791862 Admin users should be able to revoke admin users

#### What does this PR do?
- It enables admin users to be able to revoke admin users

#### Description of Task to be completed?
- create revoke admin confirmation modal

- create action, reducers to enable admin to revoke admin users

- test functionality
#### How should this be manually tested?
- Log into the application as an admin.
- Access the Admin Panel, navigate to the Manage Users > click the revoke button for a particular admin user.
- A modal will pop up then confirm the revoke action
- You should get a success message

#### What are the relevant pivotal tracker stories?
- [#166791862](https://www.pivotaltracker.com/story/show/166791862)

#### Background Context

#### Screenshots (If Applicable)
